pinot-core/src/main/java/org/apache/pinot/core/segment/virtualcolumn/DefaultNullValueSingleStringVirtualColumnProvider.java

```diff
@@ -19,11 +19,10 @@
 package org.apache.pinot.core.segment.virtualcolumn;
 
 /**
- * Virtual column provider that returns the current host name.
+ * Provide the default null value as a single string.
  */
-public class HostNameVirtualColumnProvider extends SingleStringVirtualColumnProvider {
-  @Override
+public class DefaultNullValueSingleStringVirtualColumnProvider extends SingleStringVirtualColumnProvider {
```

Comment:
This class is good for this PR. In your following PRs, you might want to change this class into a general DefaultNullValueVirtualColumnProvider

Reply:
+1